tests/__snapshots__/compile.test.ts.snap

```diff
@@ -43,22 +149,22 @@ exports[`literals.ts 1`] = `
   .name	"foo"
   .pname	p1, "v"
   set_reg	5, A
-  check_battery	nil, A
```

Comment:
I'm not sure if it was intentional that `nil` was emitted here. This changed when I fixed some issues with the assembler effected blueprints and the result seems OK?

Reply:
I think both versions make sense. The .ts code isn't doing a jump so I can see why outputting nil corresponds with that, but adding a label is more clear.